src/Microsoft.AspNet.Testing/xunit/EnvironmentVariableSkipConditionAttribute.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.Testing.xunit;
+
+namespace Microsoft.AspNet.Testing.xunit
+{
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = false)]
+    public class EnvironmentVariableSkipConditionAttribute : Attribute, ITestCondition
+    {
+        private readonly string _environmentVariableName;
+        private readonly string _environmentVariableValue;
+
+        public EnvironmentVariableSkipConditionAttribute(string environmentVariableName, string environmentVariableValue)
```

Comment:
wrap the line. it is suggested to make the code more readable in github code review setting.